roles/virtualenv3/tasks/main.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: make sure we have python3.6
+  pkgng:
+    name: "{{item}}"
+    state: present
+  with_items: 
+    - python36
+    - py27-virtualenv
+  environment: "{{ proxy_env }}"
+
+- name: install system packages
+  pkgng:
+    name: "{{item}}"
+    state: present
+  with_items: "{{ venv_packages }}"
+  environment: "{{ proxy_env }}"
+
+- name: create the virtual environment
+  become: yes
+  become_user: "{{ venv_user }}"
+  pip:
+    name: "{{ item }}"
+    editable: false
+    virtualenv: "{{ venv_home_dir }}/{{ venv_name }}"
+    virtualenv_python: "python3.6"
```

Comment:
There's also a `virtualenv_command` option where you can pick `virtualenv-3.6`. Doing that automatically selects python 3.6 for the virtualenv.

Reply:
yeah, I did not try this path yet